pkg/config/config_test.go

```diff
@@ -157,6 +157,14 @@ fDGt+yaf3RaZbVwHSVLzxiXGsu1WQJde3uJeNh5c6z+5
 			},
 			"*backends.LocalSecretManager",
 		},
+		{
```

Comment:
Can you also add a case for a using the one password backend with a missing variable in the TestNewConfigMissingParameter test

Reply:
Sure

Reply:
Just realized I neglected to push the additional test I wrote to the correct remote. Fixed now.